docker/via-external-node/Dockerfile

```
@@ -1,6 +1,8 @@
 # Will work locally only after prior contracts build
 
-FROM matterlabs/zksync-build-base:latest AS builder
+FROM lukemathwalker/cargo-chef:0.1.77-rust-bookworm AS cargo-chef
+
+FROM matterlabs/zksync-build-base:latest AS chef
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)